### DISCUSSION TITLE
Chrome OS.

### DISCUSSION THREAD

**Discussion by Canaanabolaanan** [powertab/powertabeditor#319] [Ideas]
Aaack. I'm learning how to use the new discussion feature. I'll post an official thread here. I see all these wonderful updates to the program, but alas, I only have access presently to Chrome OS. Would love to be able to check it out. Cameron sent me a link to use Ubuntu to install it on, so that's evidently a viable workaround for those with compatible chrome OS devices (and I'll check it out soon) but I just thought I'd put it to the community to see if anyone might want to adapt this to the OS directly. Maybe it's not a terribly popular software and so would be more work than worth. Thanks for the consideration :) 

Link for those interested - https://www.androidpolice.com/2018/08/19/install-linux-applications-chrome-os/

**Comment by cameronwhite** [comment]
Looking into it a bit more, a ChromeOS / Android build would probably be possible, in the same manner as Krita (which also uses Qt and Boost)
See https://krita.org/en/item/first-krita-beta-for-android-and-chromeos-in-play-store/ and https://github.com/KDE/krita/tree/master/packaging/android
Implementing an Android / NDK backend for RtMidi would probably also be required, though

So I think the summary is that it seems viable with a moderate amount of work, but getting the Linux build running on ChromeOS would be the most immediate solution.
